apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/schema/SchemaManager.java

```diff
@@ -632,7 +632,14 @@ private Uni<Schema.CqlKeyspaceDescribe> getKeyspaceInternal(
                 // if it's not null, we map to keyspace with cached flag true
                 return Uni.createFrom()
                     .future(keyspaceFuture)
-                    .map(
+
+                    // if failure is cached, invalidate
+                    .onFailure()
```

Comment:
Does this change some functionality?

Reply:
No, tests remained the same.. They changed how failures are cached and this we need to invalidate if we get a failure emitted from the cache.. everything should be the same as before..